services/portfolio_view.py

```
@@ -1055,6 +1084,14 @@ def __init__(self, *, snapshot_backend: Any | None = None) -> None:
         self._dataset_cache_adapter: PortfolioDatasetCacheAdapter | None = (
             _get_dataset_cache_adapter()
         )
+        if self._dataset_cache_adapter is not None:
+            try:
+                self._dataset_cache_adapter.clear()
+            except Exception:  # pragma: no cover - defensive safeguard
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)